Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
```

Comment:
This file has been formatted bu rustfmt..
If you want me to rollback let me know but the old version has some inconsistencies.